docs/getting-started/guides/how-to-understand-strategies-descriptions.md

```diff
@@ -0,0 +1,111 @@
+# How to Understand Strategies Descriptions
+
+Navigating strategies descriptions at [yearn.watch](https://yearn.watch/) or [vaults.yearn.finance](https://vaults.yearn.finance/) may expose you to terms that you have never seen, this page contains a complete glossary on what each term means when used to describe strategies for yearn vaults.
+
+It complements our DeFi [Glossary](../../resources/defi-glossary) and it's more focused only on the words used by strategy descriptions and shorter explanations.
+
+- Regular links navigate between the glossary.
+- Links marked with ↗ go to external websites.
+
+# Yearn
+
+### yVault / Yearn Vault
+- [Smart contract↗](https://ethereum.org/en/developers/docs/smart-contracts/) that holds user deposits and allocates into different strategies (up to 20 in the current version 2).
+### yvToken / Token
+- Token is any [ERC20-compatible↗](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/) DeFi token.
+- Yearn Vault Token (yvToken) is an ERC20-compatible token that represents a [position](#position).
+- For example: yvDAI represents a position in the DAI Yearn Vault.
+### Strategy
+- A [smart contract↗](https://ethereum.org/en/developers/docs/smart-contracts/) that knows how to allocate funds into a specific [yield](#yield--profit)-generating strategy.
+### Strategist
+- A person that creates and maintains one or more strategies.
+
+# Finance
+
+### Yield / Profit
+- Money (or tokens) made by an investment that surpasses the invested initial amount.
+### Interest
+- Fee charged for [lending](#lend--borrow-against) money.
+### Compound / Autocompound
+- Compound is when you deposit yields into the same investment that it came from.
```

Comment:
This seems confusing to me. Something around `turning the yield into principal` might sound more clear. I don't have a strong feeling about this one

Reply:
can you check the new wording to see if it's better?